charts/osm/templates/cleanup-hook.yaml

```diff
@@ -16,6 +16,9 @@ spec:
         {{- include "osm.labels" . | nindent 8 }}
     spec:
       serviceAccountName: {{ .Release.Name }}-cleanup
+      nodeSelector:
+        kubernetes.io/arch: amd64
+        kubernetes.io/os: linux
```

Comment:
Already done in https://github.com/openservicemesh/osm/pull/3622/files, present on L23.